tools/launch_uv4.bat

```diff
@@ -0,0 +1,58 @@
+::
```

Comment:
The crazy thing is the MDKv5 has uv4.exe in a UV4 directory! But good idea. I'll rename the script